<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compare_window</class>
 <widget class="QMainWindow" name="compare_window">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plan comparison</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="compare_widget">
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(211, 215, 207);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
   </property>
   <widget class="QPushButton" name="close_info_pbutton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>410</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="baseline_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>390</width>
      <height>346</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid;
border-color: rgb(211, 215, 207);</string>
    </property>
    <layout class="QVBoxLayout" name="baseline_meta_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="baseline_layout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QScrollBar" name="slice_selection_sbar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>571</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>239</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="value">
     <number>49</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="reference_widget" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>390</width>
      <height>346</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid;
border-color: rgb(211, 215, 207);</string>
    </property>
    <layout class="QVBoxLayout" name="reference_meta_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="reference_layout">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="baseline_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="reference_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="slice_selection_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>118</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Slice selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="opacity_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>Dose opacity</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="opacity_sbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);
margin-right: 6px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QLabel" name="slice_selection_pos">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>380</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
